### PR TITLE
JmapAllSimulation recipients feeder should be infinite

### PR DESCRIPTION
**Pull request by blackheaven**

### CODE REVIEW

**Review by remk — APPROVED** [review]

**Review by rouazana — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

File: src/test/scala-2.12/org/apache/james/gatling/simulation/jmap/JmapAllSimulation.scala [hunk 1]
```diff
@@ -19,7 +19,7 @@ class JmapAllSimulation extends Simulation {
 
   private val scenario = new JmapAllScenario()
 
-  setUp(scenario.generate(UserFeeder.toFeeder(users), Configuration.ScenarioDuration, RecipientFeeder.usersToFeeder(users))
+  setUp(scenario.generate(UserFeeder.toFeeder(users), Configuration.ScenarioDuration, RecipientFeeder.usersToFeeder(users).random)
```
Comment (mbaechler) [inline src/test/scala-2.12/org/apache/james/gatling/simulation/jmap/JmapAllSimulation.scala]: is there any case where we want a finite feeder? If not, we could put the random into usersToFeeder

**Review by chibenwa — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Review by chibenwa — APPROVED** [review]

File: src/main/scala-2.12/org/apache/james/gatling/control/RecipientFeeder.scala [hunk 2]
```diff
@@ -18,10 +19,11 @@ object RecipientFeeder {
...
-  def usersToFeeder(users: Seq[User]): RecipientFeeder =
+  def usersToFeeder(users: Seq[User]): RecipientFeederBuilder =
     users
       .map(user =>
         Map(
           recipientSessionParam -> user.username.value))
       .toArray
-}
+      .random
+  }
```
Comment (mbaechler) [inline src/main/scala-2.12/org/apache/james/gatling/control/RecipientFeeder.scala]: is it an accidental indent or does it make sense? (hard to figure out in github)

Comment (blackheaven) [reply src/main/scala-2.12/org/apache/james/gatling/control/RecipientFeeder.scala]: it is supposed to be aligned with `.toArray`

Comment (mbaechler) [reply src/main/scala-2.12/org/apache/james/gatling/control/RecipientFeeder.scala]: I was asking for the closing curly brace

**Review by Arsnael — APPROVED** [review]

**Review by mbaechler — APPROVED** [review]

**Review by chibenwa — APPROVED** [review]

**Pull request merged**